libstuff/SRandom.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+#include <random>
+using namespace std;
```

Comment:
The need for this line is a great reason why splitting`libstuff` up into separate files just creates complexity for no gain.  LIbstuff is a single homogenous library that is included all or nothing.  The way you use libstuff is you include `libstuff.h` as the very first include of every `.cpp` file; never "pick and choose" to include subsets.

So, remove the `using namespace std;` above because it's unnecessary if `libstuff.h` is being included correctly (eg, the very first include), and instead just update `SRandom.cpp` to include `libstuff.h` at the top (without including `SRandom.h` at all).